### PR TITLE
N8N-2556 correct binding for backspace

### PR DESCRIPTION
**Pull request by saintsebastian**
Linear reference [N8N-2556](https://linear.app/n8n/issue/N8N-2556/bug-deactivating-workflow-when-there-is-an-error-in-the-current)

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -568,9 +568,11 @@ export default mixins(
 					return;
 				}
 
+				// TODO add delete/backspace
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:571]: please remove this comment

**Review by mutdmour — APPROVED** [review]
LGTM. Please remove comment and should be good to go

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.154.0`

**Pull request merged**